workshops/read_the_internet/README.md

```diff
@@ -0,0 +1,175 @@
+---
+name: 'Reading the Internet'
+description: 'Use Python to read HTML and extract any information you can find!'
+author: '@RafaelCenzano'
+img: 'https://cloud-p5pdve0kf.vercel.app/2webscrape.jpg'
+---
+
+We can read all the content on websites, but wouldn't it be cool if we could automatically read specific data from websites whenever you wanted? If this interested you, this workshop is for you. I will be showing you how to use webscraping in Python.
+
+*Note: not all webpages can be read using this technique. If data is loaded in using ajax or after the inital page request this method may not work.*
+
+[Final Result and Code](https://repl.it/@SavageCoder77/Read-The-Internet#main.py)
+
+## Prerequisites
+You should have a basic understanding of **HTML** and **Python** and how to use variables and objects.
+
+## Setup
+![Lets do it gif](https://cloud-p23nck0rj.vercel.app/1lets-start.gif)
+
+We will be using [repl.it](https://repl.it), an online code editor that allows you to write and run code from any computer!
+
+Spin up a Python3 repl by [clicking here](https://repl.it/languages/python3).
+
+Before we can start coding, we need to install a couple of external libraries. Follow the image below to find where to add the packages. Once in this menu you will need to install `beautifulsoup4`, `requests`, and `lxml`.
+
+![Repl.it packages menu](https://cloud-p5pdve0kf.vercel.app/0ezgif.com-video-to-gif.gif)
+
+#### What each package does
+`lxml` is a C package used by beautiful soup that parses the HTML, `beautifulsoup4` is for working with the parsed HTML, and `requests` is for requesting the webpage/html.
+
+<details>
+
+<summary> Learn more about the packages: </summary>
+
+- [BS4](https://www.crummy.com/software/BeautifulSoup/bs4/doc/)
+- [requests](https://requests.readthedocs.io/en/master/)
+- [LXML](https://lxml.de) 
+
+</details>
+
+## Lets start coding!
+
+![Start coding gif](https://cloud-5bmb4t5tp.vercel.app/1start-coding.gif)
+
+Once you've installed the packages, open the `main.py` file in the sidebar on the left. Then, import the packages you just installed.
+
+```python
+from bs4 import BeautifulSoup as bs
+import requests
+```
+
+Now we can start webscraping! We will begin by requesting our webpage and extracting the HTML from the request object.
+
+```python
+url = 'https://ncov2019.live/data'
+# request the website
+r = requests.get(url)
+# get the HTML from the request
+page = r.text
+```
+
+Next we will create the BeautifulSoup object and use `lxml` to parse the HTML.
+
+```python
+soup = bs(page, 'lxml')
+```
+
+Now that we have our BeautifulSoup object we can go all of the HTML! 
+
+### Working with the parsed HTML
+
+![HTML gif](https://cloud-5bmb4t5tp.vercel.app/2html.gif)
+
+In the next step, we'll be working with data from an HTML table. So, before, we continue, a quick mini-lesson on HTML tables:
+
```

Comment:
I think in this step it should be a lot more clear why you're suddenly learning about HTML tables. When I first read this it was a little jarring because I was like "wait what? i thought we were writing python. Why are we suddenly pausing to learn about HTML tables?"